.mvn/jvm.config

```diff
@@ -1 +0,0 @@
--Xmx800m
```

Comment:
why?

Reply:
committed in error (my local machine can't build with that in there for some reason)
I've replaced the commit